bin/node-template/pallets/template/src/benchmarking.rs

```diff
@@ -1,6 +1,7 @@
 //! Benchmarking setup for pallet-template
 #![cfg(feature = "runtime-benchmarks")]
 use super::*;
+use sp_std::vec;
```

Comment:
This looks wrong? I don't see any usage of the macro in here. Maybe some macro needs this? If yes, it should pull this in automatically.

Reply:
It is expected here https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/3434676
```
[2023-08-24 12:21:48]   error: cannot find macro `vec` in this scope
[2023-08-24 12:21:48]     --> /builds/parity/mirrors/substrate/bin/node-template/pallets/template/src/benchmarking.rs:10:1
[2023-08-24 12:21:48]      |
[2023-08-24 12:21:48]   10 | #[benchmarks]
[2023-08-24 12:21:48]      | ^^^^^^^^^^^^^
[2023-08-24 12:21:48]      |
[2023-08-24 12:21:48]      = help: consider importing one of these items:
[2023-08-24 12:21:48]              crate::benchmarking::__private::vec
[2023-08-24 12:21:48]              frame_benchmarking::__private::vec
[2023-08-24 12:21:48]              scale_info::prelude::vec
[2023-08-24 12:21:48]              sp_std::vec
[2023-08-24 12:21:48]      = note: this error originates in the attribute macro `benchmarks` (in Nightly builds, run with -Z macro-backtrace for more info)
```